cmd/new.go

```diff
@@ -167,10 +167,15 @@ func createAndEditSnippet(newSnippet snippet.SnippetInfo, snippets snippet.Snipp
 
 func countSnippetLines() int {
 	// Count lines in snippet file
-	f, err := os.Open(config.Conf.General.SnippetFile)
+	if config.Conf.General.SnippetFile == "" {
```

Comment:
It might be nicer to handle the empty string case inside ExpandPath itself. 

No validation is taking place there, so right now the function ExpandPath is kind of buggy. Adding this empty string check there would make it more robust, and also make this code a bit nicer to read here.

Reply:
That's valid. Let me give it a go.